chainercv/links/model/resnet/resnet.py

```diff
@@ -191,6 +198,13 @@ def __init__(self, n_layer,
             self.prob = F.softmax
 
         if pretrained_model in _models:
+            if not _available:
+                warnings.warn('cv2 is not installed on your environment. '
+                              'The scores of ResNets reported in the '
+                              'ChainerCV\'s Github page are calculated using '
```

Comment:
`Github` -> `GitHub`.

Reply:
`Github page` sounds that we have `***.github.io` page.

Reply:
Why did you change the warning message from that of SSD? It is better to use the same message because they say the same thing.

Reply:
I see. I understand the difference.